polaris-react/src/components/IndexFilters/IndexFilters.tsx

```diff
@@ -97,6 +103,8 @@ export interface IndexFiltersProps
   closeOnChildOverlayClick?: boolean;
   /** Optional override to the default keyboard shortcuts available */
   disableKeyboardShortcuts?: boolean;
+  /** Shows the edit columns button  */
```

Comment:
I think the prop name is fine, but could we make this description more helpful? I think there could be a confusion that this would override the "Edit columns" option in the Popover when clicking on a view, which it wouldn't. 
```suggestion
  /** Shows the edit columns button as a filter button when the IndexFilters is in Default mode  */
```